person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+
+  public
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
```

Comment:
- [ ] **[OPTIONAL]** Instead defining `private` and `public`, it is often the convention that private methods are put at the bottom of the class, so there is no need to define `private` and then `public` methods again.

  ```
  def can_use_services?
  ...

  private

  def of_age?
  ...
  ```